### PR TITLE
Resident search Postgres tech notes

### PR DESCRIPTION
**Pull request by tony-griffin**

### CODE REVIEW

File: docs/runbooks/search-functionality/resident-search.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+## Technical Notes
+
+### Postgres
+`pg_tgrm extension for Postgres`
+
+- The person search uses fuzzy name matching that uses the postgres trigram extension in queries. 
+- The *pg_tgrm* extension adds required functionality and also creates a new index to improve performance.
+- If the extension is not installed in your (table/database?) you will get the following error:
```
Comment (rachel-hutchinson) [inline docs/runbooks/search-functionality/resident-search.md:14]: Suggestion: If the extension is not installed in your database you will get the following error when you try to run queries that make use of trigrams:

**Review by rachel-hutchinson — COMMENTED** [review]
This looks good, I've suggested a change
I was also wondering if it might be good to expand on the overview of how the search functionality currently searches - i.e. it does fuzzy name searching but date of birth and postcode searching is strict and if a person id is searched for that is the only search term that will be used
Another thing I was wondering was would it be useful to have the SQL query in this document? Because within the API code it is split into different parts although it's still readable within the code itself

File: docs/runbooks/search-functionality/resident-search.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+id: resident-search
+title: Resident Search Configuration
+slug: /runbooks/search-functionality/resident-search
```
Comment (acodeninja) [inline docs/runbooks/search-functionality/resident-search.md:4]: This isn't in the correct runbook, it should be under `/runbooks/case-viewer` as this functionality is part of that application.

File: docs/runbooks/search-functionality/resident-search.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+The following are all string types and optional
+
+`name`
+`date_of_birth`
+`postcode`
+`person_id`
+`cursor`
+
+## Jira Tickets
+Proposed SQL queries: https://hackney.atlassian.net/browse/SCT-1735
```
Comment (acodeninja) [inline docs/runbooks/search-functionality/resident-search.md:85]: I would avoid referencing a Jira instance here.

File: docs/runbooks/search-functionality/resident-search.md (hunk 2 above)
Comment (tony-griffin) [reply docs/runbooks/search-functionality/resident-search.md:4]: Thanks, I'll move that

File: docs/runbooks/search-functionality/resident-search.md (hunk 1 above)
Comment (tony-griffin) [reply docs/runbooks/search-functionality/resident-search.md:14]: Sounds good thanks

**Review by acodeninja — APPROVED** [review]
Looks good to me to merge 👍

File: docs/runbooks/search-functionality/resident-search.md (hunk 3 above)
Comment (tony-griffin) [reply docs/runbooks/search-functionality/resident-search.md]: Removed

**Pull request merged**